src/js/gi/containers/LandingPageForm.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import AboutYourselfTeaser from '../components/AboutYourselfTeaser';
+import AboutYourselfFields from '../components/AboutYourselfFields';
+import AboutYourSchoolFields from '../components/AboutYourSchoolFields';
+import AdditionalResources from '../components/AdditionalResources';
+
+class LandingPageForm extends React.Component {
+
+  render() {
+    return (
+      <div className="row">
+        <form action="/gi-bill-comparison-tool/institutions/search">
```

Comment:
Does this end up doing a full page refresh? If it's hooked up in the routes, you should have the `onSubmit` handler do something like a `history.push` instead, so it can render in place.